tensorflow/core/ops/math_ops.cc

```diff
@@ -996,7 +996,41 @@ output: Has same shape as data, except for dimension 0 which
 has size `num_segments`.
 
 )doc");
+REGISTER_OP("UnsortedSegmentMax")
+    .Input("data: T")
+    .Input("segment_ids: Tindices")
+    .Input("num_segments: int32")
+    .Output("output: T")
+    .Attr("T: realnumbertype")
+    .Attr("Tindices: {int32,int64}")
+    .Doc(R"doc(
+Computes the Max along segments of a tensor.
+
+Read [the section on
+Segmentation](../../api_docs/python/math_ops.md#segmentation) for an explanation
+of segments.
```

Comment:
Instead of copying the full comment, perhaps point to the comments for UnsortedSegmentSum and mention the differences.